api/docs/release.dox

```diff
@@ -239,6 +239,8 @@ Further non-compatibility-affecting changes include:
    with one destination and six sources.
  - Added a new label to help in handling "rseq" (Linux restartable sequence) regions:
    #DR_NOTE_RSEQ_ENTRY.
+ - Added a new API for getting the offset of an instr in an instrlist that has been
```

Comment:
Name the new API routine here so the user knows what it is and so Doxygen will add an html link (be sure to include empty parentheses to get the link).

Reply:
Done.